src/forecast.py

```diff
@@ -213,7 +199,108 @@ def forecast_page():
             )
         )
 
-    st.plotly_chart(fig, theme="streamlit")
+        start_datetime_utc, end_datetime_utc = determine_start_and_end_datetimes(start_datetimes, end_datetimes)
+
+        if start_datetime_utc and end_datetime_utc:
+            if start_datetime_utc < end_datetime_utc:
+                # Initialize Elexon API client
```

Comment:
do you need these two if statements? Wouldnt they be better dealt with in `determine_start_and_end_datetimes`